server/datastore/migrate/migrate.go

```diff
@@ -110,6 +127,13 @@ var repoTokenUpdate = `
 UPDATE repos SET repo_token = '';
 `
 
+var repoBuildNumberColumn = `
+ALTER TABLE repos ADD COLUMN repo_build_number INTEGER
```

Comment:
Is there any chance that two builds from the same repository could be running at the same time? If so, two builds could receive the same build number. Unfortunately I believe my current solution also has this issue.